contrib/build-wine/prepare-wine.sh

```diff
@@ -11,6 +12,30 @@ export WINEPREFIX=/opt/wine64
 PYHOME=c:/python$PYTHON_VERSION
 PYTHON="wine $PYHOME/python.exe -OO -B"
 
+
+# based on https://superuser.com/questions/497940/script-to-verify-a-signature-with-gpg
+verify_signature() {
+    local file=$1 keyring=$2 out=
+    if out=$(gpg --no-default-keyring --keyring "$keyring" --status-fd 1 --verify "$file" 2>/dev/null) &&
+       echo "$out" | grep -qs "^\[GNUPG:\] VALIDSIG "; then
```

Comment:
This doesn't work on my machine as my locale is JA, and the output is all in Japanese.

Can there be an alternative method?

Reply:
perhaps run gpg command with `LANG=en_US.UTF-8 ` before it? @SomberNight @ecdsa

Reply:
(If you run with LANG=ja_JP.UTF-8 you can see what I see)